cypress/integration/admin.spec.js

```diff
@@ -0,0 +1,44 @@
+const { join } = require('path');
+
+context('Admin interface', () => {
+  it('should redirect anonymous users to the login page', function() {
+    cy.visit('/admin')
+    cy.location().should('match', /\/admin\/login\//)
```

Comment:
Just a little more readable.

```suggestion
    cy.location().should(`contain`, `/admin/login`)
```